routes/dashboard.ts

```diff
@@ -38,6 +60,17 @@ export default (pool: mt.Pool, _log: Debugger): express.Router => {
         res.redirect('/icon.png');
     });
 
+    router.post('/save-sort', async (req: express.Request<any, any, SaveSortRequestBody>, res: express.Response) => {

```

Comment:
I don't think storing the sort oder in the cookies makes much sense. Why not just get it from the query parameters in `/`?